views/administration/users_edit.hbs

```diff
@@ -120,36 +120,40 @@
                         {{/unless}}
                     {{else}}
                     <div class="col-md-6">
-                        <input name="consentId" type="hidden" value="{{@root.consent._id}}" disabled />
+                        <input name="consentId" type="hidden" value="{{consent._id}}" disabled />
 
                         <label><input name="form" value="analog" type="radio" disabled
-                                {{#ifeq @root.consent.userConsent.form 'analog'}}checked{{/ifeq}} />
+                                {{#ifeq consent.userConsent.form 'analog'}}checked{{/ifeq}} />
                             {{$t "administration.label.analog" }}</label>
                         <label><input name="form" value="digital" type="radio" disabled
-                                {{#ifeq @root.consent.userConsent.form 'digital'}}checked{{/ifeq}} />
+                                {{#ifeq consent.userConsent.form 'digital'}}checked{{/ifeq}} />
                             {{$t "administration.label.digital" }}</label>
                         <br />
                         <label><input name="privacyConsent" type="checkbox" value="true" disabled
-                                {{#if @root.consent.userConsent.privacyConsent}}checked{{/if}} />
+                                {{#if consent.userConsent.privacyConsent}}checked{{/if}} />
                             {{$t "administration.global.label.dataProtection" }}</label><br />
                         <label><input name="termsOfUseConsent" type="checkbox" value="true" disabled
-                                {{#if @root.consent.userConsent.termsOfUseConsent}}checked{{/if}} />
+                                {{#if consent.userConsent.termsOfUseConsent}}checked{{/if}} />
                             {{$t "administration.global.label.termsOfUse" }}</label><br />
                     </div>
                     {{/unless}}
                 </div>
             {{else}}
-                <h2 class="h5">{{$t "administration.button.onlyRegistrationNeeded" }} {{{ @root.consentStatusIcon}}}</h2>
-                {{#unless @root.hasAccount}}
+                <h2 class="h5">{{$t "administration.button.onlyRegistrationNeeded" }} {{{consentStatusIcon}}}</h2>
+                {{#unless hasAccount}}
                 <div class="row" id="consents-overview">
                     <div class="col-md-6">
                         <p>
-                            {{$t "administration.longText.text.registrationNotCompleted" }}
+                            {{#unless editTeacher}}
```

Comment:
the separation of cases due to "editTeacher" permission is unclear to me, I though this text should not be shown in the case you are with instances? maybe I missunderstood. also if the "else" in Line 149 regards to the unless in line 147, where is the difference in this cases?

Reply:
that handlebar file handles both cases: editing students and editing teachers. in case of a editing teacher - and that teacher did not finish the registration process - we just show another info message then for students. hope I got your comment right - otherwise lets talk

Reply:
ah now I got it, yes thats fine for me, thx 👍